src/main/java/com/libraryman_api/borrowing/BorrowingService.java

```diff
@@ -264,14 +271,19 @@ private BigDecimal calculateFineAmount(Borrowings borrowing) {
      * @param memberId the ID of the member whose borrowings are to be retrieved
      * @param pageable the pagination information, including the page number and size
      * @throws ResourceNotFoundException if the member has not borrowed any books
+     * @throws InvalidSortFieldException if an invalid sortBy field is specified
      * @return a {@link Page} of {@link Borrowings} representing all borrowing associated with a specific member
      */
-    public Page<Borrowings> getAllBorrowingsOfMember(int memberId, Pageable pageable) {        
-        Page<Borrowings> borrowings = borrowingRepository.findByMember_memberId(memberId, pageable);
-        
-        if (borrowings.isEmpty()) {
-            throw new ResourceNotFoundException("Member didn't borrow any book");
+    public Page<Borrowings> getAllBorrowingsOfMember(int memberId, Pageable pageable) {                
```

Comment:
What sorting parameters can be passed here?

Reply:
For **Borrowings**, all fields in the `Borrowings` model (e.g., borrowingId, etc.) are available for sorting. If you need to sort by nested fields (like Member.name), use the format `fieldName_propertyName` (e.g., member_name, book_name).